backend/api/contacts/src/main/java/co/airy/core/contacts/ContactsController.java

```diff
@@ -106,8 +107,16 @@ public ResponseEntity<?> importContacts(@RequestBody @Valid List<CreateContactPa
     public ResponseEntity<?> listContacts(@Valid @RequestBody(required = false) ListContactsRequestPayload payload) {
         payload = payload == null ? new ListContactsRequestPayload() : payload;
         final List<Contact> contacts = stores.getAllContacts();
-
-        Paginator<Contact> paginator = new Paginator<>(contacts, Contact::getId)
+        final List<Contact> sortedContacts = contacts.stream()
+                .filter(c -> c.getDisplayName() != null)
+                .sorted(comparing(Contact::getDisplayName))
+                .collect(toList());
+        final List<Contact> noDisplayNameContacts = contacts.stream()
+                .filter(c -> c.getDisplayName() == null)
+                .collect(toList());
+        sortedContacts.addAll(noDisplayNameContacts);
```

Comment:
So I looked this up and instead of filtering and adding you can also wrap the comparator in `Comparator.nullsLast(comparing(Contact::getDisplayName))` [docs](https://docs.oracle.com/javase/8/docs/api/java/util/Comparator.html#nullsLast) which should be more efficient

Reply:
Oh thanks @chrismatix I wasn't aware of this function